src/main/java/linkedList/LinkedList.java

```diff
@@ -0,0 +1,93 @@
+package linkedList;
+
+public class LinkedList<T> {
+    Node head;
+    Node tail;
+    int currentIndex = 0;
+
+    public void insert(T data) {
+        Node node = new Node();
+        node.data = data;
+        node.next = null;
+        node.prev = null;
+        if (head == null) {
+            head = node;
+            tail = node;
+        } else {
+            Node n = head;
+            while (n.next != null) {
+                n = n.next;
+            }
+            node.prev = n;
+            n.next = node;
+            tail = n.next;
+        }
+    }
+
+    public void deleteAt(int index) {
+        if (index == 0) {
+            head = head.next;
+            head.prev = null;
+        } else {
+            Node previousNode = head;
+            Node nodeToDelete;
+            Node nextNode;
+            for (int i = 0; i < index - 1; i++) {
```

Comment:
it will fail if too big `index` is passed to a list that contains few elements.
https://docs.oracle.com/javase/8/docs/api/java/lang/ArrayIndexOutOfBoundsException.html :)